app/(root)/admin/page.tsx

```
@@ -0,0 +1,103 @@
+"use client";
+
+import QuestCard from "@/components/QuestCard";
```

ESLint

[error] 'QuestCard' is defined but never used.

(@typescript-eslint/no-unused-vars)